### ISSUE TITLE
Addition of Lm_sensors option in Computer Stats setting

### ISSUE THREAD

**Issue by EZarnosky** [Hellowlol/HTPC-Manager#226]
It looks like Open Hardware Monitor is kinda dead, no real updates in quite some time. Added to this, it is a mono application... mono adds bloat to samll installs.

Any chance of adding Lm_sensors as a replacement/alternate option to OHM?

**Comment by Hellowlol** [comment]
Tldr: no :) 

Longer:
I think ohm is awesome. I don't know if there is a other program that does what it does, have a webserver and is available on as many platforms. Last update was 6 months ago, lm sensors had 1,5 years between the two latest releases.

I do see your point having to install mono, but I think that is a small price to pay. There several good programs to monitor/find hardware info for Linux out there. Mono is also getting more common because of cool programs like sonarr. Rewriting this would be major pain, I don't have time nor interest (my main platform is windows)

I will accept a pr, so if anybody out there is willing to do the work I'll merge it.

**Comment by EZarnosky** [comment]
You should get a Raspberry/Banana Pi. Makes a hell of a headless NZB/Torrent download server.

Mine does all the downloading and pushes it up to my NAS. :)

**Comment by Hellowlol** [comment]
I have two. :p
